_includes/navbar.html

```diff
@@ -19,10 +19,11 @@
       </div>
       <ul class="usa-button-list usa-unstyled-list">
         <li>
-          <a class="usa-button" href="#">Download</a>
+          <a class="usa-button" href="#" onclick="ga('send', 'event', 'Downloaded framework', 'Clicked download button inside site');">Download</a>
```

Comment:
The only comment I'll make about this is that these would be better in `data-` attributes which get parsed more globally with jquery. But This is fine if we want to keep things moving.